src/ml_tooling/storage/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .file import FileStorage
 from .base import Storage
+from .artifactory import ArtifactoryStorage
```

Comment:
`ArtifactoryStorage` can't be in init. At least not with out try-catch statement as some users won't have it installed. That is why Travis is failing

Reply:
Changed to be lazy-loading - first throws exeception if artifactory is not installed when trying to instantiate Artifactory Storage